tests/manual-test-cases/Group5-Functional-Tests/5-29-Opaque-Network.robot

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 5-29 - Opaque Network
+Resource  ../../resources/Util.robot
+Suite Setup  Vdnet NSXT Topology Setup
+Suite Teardown  Vdnet NSXT Topology Cleanup  ${NIMBUS_POD}  ${testrunid}
+
+*** Variables ***
+${NIMBUS_LOCATION}  sc
+${VDNET_LAUNCHER_HOST}  10.92.34.93
+${USE_LOCAL_TOOLCHAIN}  0
+${VDNET_MC_SETUP}  0
+${NIMBUS_BASE}  /mts/git
+${NSX_VERSION}  ob-10536046
+${VC_VERSION}  ob-7312210
+${ESX_VERSION}  ob-7389243
+
+${nsxt_username}  admin
+${nsxt_password}  Admin\!23Admin
+
+*** Keywords ***
+Vdnet NSXT Topology Setup
+    [Timeout]    180 minutes
+    ${TESTRUNID}=  Evaluate  'NSXT' + str(random.randint(1000,9999))  modules=random
+    ${json}=  OperatingSystem.Get File  tests/resources/nimbus-testbeds/vic-vdnet-nsxt.json
+    ${file}=  Replace Variables  ${json}
+    Create File  /tmp/vic-vdnet-nsxt.json  ${file}
+
+    Open Connection  ${VDNET_LAUNCHER_HOST}
+    Wait Until Keyword Succeeds  2 min  30 sec  Login  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}
+
+    Put File  /tmp/vic-vdnet-nsxt.json  destination=/tmp/vic-vdnet-nsxt.json
+    #Deploy the testbed
+    ${result}=  Execute Command  cd /src/nsx-qe/automation && VDNET_MC_SETUP=${VDNET_MC_SETUP} USE_LOCAL_TOOLCHAIN=${USE_LOCAL_TOOLCHAIN} NIMBUS_BASE=${NIMBUS_BASE} vdnet3/test.py --config /tmp/vic-vdnet-nsxt.json /src/nsx-qe/automation/TDS/NSXTransformers/Openstack/HybridEdge/OSDeployTds.yaml::DeployOSMHTestbed --testbed save
```

Comment:
Can we assign NIMBUS_LOCATION during setup?

Reply:
It is used in json file.